examples/transformers/main.py

```diff
@@ -0,0 +1,1107 @@
+#!/usr/bin/env python
```

Comment:
Let's rename this file to run_qa.py and ideally have it be a drop in replacement for huggingface's version of that file.

Additionally, we need to add some comments at the top to turn off formatting and copyright for this file. See one of the other integration diffs for an example: https://github.com/neuralmagic/sparseml/pull/70
```
# neuralmagic: no copyright
# flake8: noqa
# fmt: off
# isort: skip_file
```

Let's be sure to call out the portions of the code that we are editing as well with comment blocks, see the above diff for an example, but in general:
    ```
####################################################################################
    # Start SparseML Integration
####################################################################################
```

Finally let's make sure the arguments we're using are consistent with what is being setup in those other integratiions